azext_edge/tests/edge/orchestration/resources/dataflow_endpoint/helper.py

```diff
@@ -0,0 +1,157 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from typing import Optional
+from unittest.mock import Mock
+
+import pytest
+import responses
+
+from azext_edge.edge.providers.orchestration.common import DATAFLOW_ENDPOINT_TYPE_SETTINGS
+from ..test_instances_unit import (
+    get_instance_endpoint,
+    get_mock_instance_record,
+)
+
+from .....generators import generate_random_string
+from ..conftest import get_base_endpoint, get_mock_resource
+
+
+def get_dataflow_endpoint_endpoint(
```

Comment:
This helper module (should it not be plural?) is not my favorite location for placement of get_dataflow_endpoint_endpoint(...) and get_mock_dataflow_endpoint_record(...). It is not consistent with other orchestration resource test modules. A pattern like this only works if you have subpackage/nested directory.

Can those functions at least be moved to a new conftest.py under dataflow_endpoint?